python_ta/reporters/html_reporter.py

```diff
@@ -84,21 +86,51 @@ def output_blob(self):
         # Date/time (24 hour time) format:
         # Generated: ShortDay. ShortMonth. PaddedDay LongYear, Hour:Min:Sec
         dt = str(datetime.now().strftime('%a. %b. %d %Y, %I:%M:%S %p'))
-        output_path = os.path.join(os.getcwd(), self.linter.config.pyta_output_file)
+
+        # Render the jinja template
+        rendered_template = template.render(date_time=dt, reporter=self)
+
+        # If a filepath was specified, write to the file
+        if self._output_filepath:
+            self._write_html_to_file(rendered_template)
+        else:
+            self._open_html_in_browser(rendered_template.encode('utf8'))
+
+    def _write_html_to_file(self, rendered_template):
+        """ Write the html file to the specified output path. """
+        output_path = os.path.join(os.getcwd(), self._output_filepath)
         with open(output_path, 'w', encoding='utf-8') as f:
-            f.write(template.render(date_time=dt,
-                                    # pyta_logo=pyta_logo_base64_encoded,
-                                    reporter=self))
-        print('Opening your report in a browser...')
-        output_url = 'file:///{}'.format(output_path)
-        webbrowser.open(output_url, new=2)
+            f.write(rendered_template)
+
+    def _open_html_in_browser(self, html, new=0, autoraise=True):
```

Comment:
For our purposes you can omit the `new` and `autoraise` arguments. We hard-coded these in the old code.